src/Pagination/Connection.hack

```diff
@@ -50,13 +50,15 @@ type PaginationArgs = shape(
  * @see https://relay.dev/graphql/connections.htm for more information about GraphQL pagination.
  * @see src/playground/UserConnection.hack for an example.
  */
+ <<GraphQL\ObjectType('Connection', 'Connection')>>
 abstract class Connection {
 
     /**
      * Node type for this connection.
      *
      * This should be the Hack class over which you want to paginate.
      */
+     <<__Enforceable>>
```

Comment:
Why does this need to be enforceable?

Reply:
if it's not enforceable I cannot do this in SqlConnection class `$node as this::TNode`
https://slack-github.com/slack/webapp/pull/128988/files#diff-eeef89f255be1dec860c0b1f779fe2eba85adfebf64bfe7e7c70ab8fef4a8703R47

Reply:
I still don’t follow. Looking at that code you shared, `$node as this::TNode` will explode whenever `this::TNode != this::TSqlNode` and is a no-op whenever `this::Node == this::TSqlNode`. No?